### ISSUE TITLE
Euro sign doesn't read as Euro in TalkBack while using accessibility in Android[Bug] 

### ISSUE THREAD

**Issue by HirakCTS** [xamarin/Xamarin.Forms#12250]
### Basic Information

- Version with issue:
- Last known good version:
- IDE:Version 8.7.7 (build 10)
- Platform Target Frameworks: 
  - iOS:  Version: 13.20.2.2 
  - Android: Version: 11.0.2.0 
  - UWP:   
- Android Support Library Version: Version: 16.7.0.13
- Nuget Packages:
- Affected Devices: Android Devices

**Comment by hartez** [comment]
@HirakCTS I am trying to reproduce this issue, but so far I cannot. Here's my attempt:

[_12251 Repro.zip](https://github.com/xamarin/Xamarin.Forms/files/5319256/_12251.Repro.zip)

I have three labels with € in them; TalkBack reads all of them to me as "Euro". Is there a detail I'm missing?

**Comment by Redth** [comment]
Since we haven't heard from you in more than 30 days, we hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that we can look into it further. Thank you!
